include/visit_struct/visit_struct.hpp

```diff
@@ -141,8 +252,7 @@ struct visitable<STRUCT_NAME, void> {
     using self_type = STRUCT_NAME;                                                                 \
     VISIT_STRUCT_PP_MAP(VISIT_STRUCT_MEMBER_HELPER_TYPE, __VA_ARGS__)                              \
   }                                                                                                \
-                                                                                                   \
-  static constexpr bool value = true;                                                              \
+  static VISIT_STRUCT_CONSTEXPR const bool value = true;                                           \
```

Comment:
I tested your version, and with fixes for `constexpr` it's working for msvc2013 :-)
Note that I had also used python to generated the macro :).
`VISIT_STRUCT_CXX11_CONSTEXPR` or similar might be good to support msvc2013.

You may close the PR.